daprdocs/content/en/operations/extending/using-grpc-components.md

```diff
@@ -0,0 +1,87 @@
+---
+type: docs
+title: "Using gRPC-based Pluggable Components"
+linkTitle: "Using gRPC-based Pluggable Components"
+weight: 250
+description: "Extending dapr with external gRPC-based components"
+---
+
+For more information on how to develop gRPC-based Pluggable Components click [here]({{< ref developing-grpc-components.md >}}).
+
+The example above shows how a state store gRPC-based pluggable component is declared and used, the same can be applied for the others components.
```

Comment:
"Above"? As this is a single document, referencing the previous document by "above" is ambiguous as there is no content or example preceding this section of the document.

I would be better to either: 
* directly link to it or 
* assume a read ordering and taking this to be the second document read, use _previous_ instead.

Reply:
this text shouldn't be there, deleted, thanks

Reply:
removed!